### PR TITLE
Setup Neeva Cookies Opt-in Backend

### PR DESCRIPTION
**Pull request by evancrow**
Added support for Neeva cookies opt-in on the Cookie Cutter side, now we just need to implement the front end.

### How was this tested?
- [x] I tested this manually

### CODE REVIEW

**Review by chung-lai — APPROVED** [review]
Awesome! LGTM. @jlynch630, please also take a look as you are more familiar with the flags :)

**Comment by chung-lai** [comment]
Please hold onto this change for now as we will have a discussion whether we want to do this.

File: Client/Frontend/Browser/ContentScripts/CookieCutterHelper.swift [hunk 1]
```diff
@@ -51,39 +51,40 @@ class CookieCutterHelper: TabContentScript {
...
+                        cookieCutterModel.optIntoNeevaCookies
+                    {
+                        // Pass false so Cookie Cutter does not block these cookies.
+                        // The user has already consented to these cookies on first run.
+                        sendResponse(
+                            data: [
+                                "cookieCutterEnabled":
+                                    TrackingPreventionConfig.trackersPreventedFor(
+                                        domain, checkCookieCutterState: true),
+                                "marketing": false,
```
Comment (jlynch630) [inline Client/Frontend/Browser/ContentScripts/CookieCutterHelper.swift:65]: The engine expects true for allowed and false for disallowed/blocked -- do you do some other transformation before these prefs reach the engine?

**Review by jlynch630 — COMMENTED** [review]
LGTM but should address the true/false debacle first

File: Client/Frontend/Browser/ContentScripts/CookieCutterHelper.swift (hunk 1 above)
Comment (evancrow) [reply Client/Frontend/Browser/ContentScripts/CookieCutterHelper.swift:65]: Well shit, I thought we had discussed that it worked similar to our ad blocker where true was passed to block the Cookie, sounds like a major miscommunication on my end. Have a hotfix to upload coming real quick.

**Comment by chung-lai** [comment]
So we decided to revisit this solution at a later time. We can close this for now and reopen later. Thanks for looking into it 🙏

**Pull request closed without merging**